syndesis/tests/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project
+xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.fusesource.ide</groupId>
+		<artifactId>syndesis</artifactId>
+		<version>10.2.0-SNAPSHOT</version>
+	</parent>
+
+	<groupId>org.fusesource.ide.syndesis</groupId>
+	<artifactId>tests</artifactId>
+	<name>JBoss Fuse Tooling :: Syndesis Support :: Tests</name>
+	<packaging>pom</packaging>
+	<properties>
+		<!-- set a different path with -Djbosstools.test.jre.7= -->
+		<jbosstools.test.jre.7>/qa/tools/opt/jdk1.7.0_last</jbosstools.test.jre.7>
```

Comment:
is it still useful? i mean, we require Java 8 now so it is surely not used.

Reply:
that has been copied from editor/tests/pom.xml...there we use it too. Is that even used any longer or maybe overridden somewhere?